src/Wallabag/ApiBundle/Tests/Controller/WallabagRestControllerTest.php

```diff
@@ -197,8 +197,8 @@ public function testPatchEntry()
         $this->client->request('PATCH', '/api/entries/'.$entry->getId().'.json', array(
             'title' => 'New awesome title',
             'tags' => 'new tag '.uniqid(),
-            'star' => true,
-            'archive' => false,
+            'starred' => '1',
+            'archive' => '0',
```

Comment:
Could you add extra tests with value like `'starred' => 1, 'archive' => 0,` without single quotes to check the behavior?